packages/metal/src/coreNamed.js

```diff
@@ -289,3 +289,48 @@ export function isString(val) {
  */
 export function nullFunction() {
 }
+
+/**
+ * Register a custom element for a given metal component.
+ *
+ * @param {String} tagName The tag name to use for this custom element.
+ * @param {!function()} ctor Metal component constructor.
+ * @param {?Array=} observedAttrs The attributes to be "observed".
+ * @return {void} Nothing.
+ */
+export function defineWebComponent(tagName, ctor, observedAttrs) {
```

Comment:
Are we doing this for performance reasons or it is that isn't there a way to get the `observedAttrs` from the module definition?